PORTAL/jobs/_workers.py

```diff
@@ -122,19 +127,21 @@ class Workers:
 
     @classmethod
     def from_config(cls, cfg, JobsFS=WorkerJobsFS):
-        worker = Worker.from_config(cfg.worker, JobsFS)
-        return cls(worker)
+        workers = {}
+        for worker_name, worker in cfg.items():
+            workers[worker_name] = Worker.from_config(worker_name, worker, JobsFS)
+        return cls(workers)
 
-    def __init__(self, worker):
-        self._worker = worker
-
-    def __repr__(self):
-        args = (f'{n}={getattr(self, "_"+n)!r}'
-                for n in 'worker'.split())
-        return f'{type(self).__name__}({"".join(args)})'
```

Comment:
FWIW, nearly every time I don't add `__repr__()` to a class, I end up wishing I had (when debugging).